redpanda/utils/utils.go

```diff
@@ -46,66 +48,66 @@ func IsNotFound(err error) bool {
 	return false
 }
 
-// StringToCloudProvider returns the controlplanev1beta1's CloudProvider code based on
+// StringToCloudProvider returns the controlplanev1beta2's CloudProvider code based on
 // the input string.
-func StringToCloudProvider(p string) (controlplanev1beta1.CloudProvider, error) {
+func StringToCloudProvider(p string) (controlplanev1beta2.CloudProvider, error) {
 	switch strings.ToLower(p) {
 	case "aws":
-		return controlplanev1beta1.CloudProvider_CLOUD_PROVIDER_AWS, nil
+		return controlplanev1beta2.CloudProvider_CLOUD_PROVIDER_AWS, nil
 	case "gcp":
-		return controlplanev1beta1.CloudProvider_CLOUD_PROVIDER_GCP, nil
+		return controlplanev1beta2.CloudProvider_CLOUD_PROVIDER_GCP, nil
```

Comment:
Azure!

Reply:
I'm kidding, it is OK to tackle on another PR.

Reply:
:saluting_face: Will do it here